admin-ui/plugins/auth-server/redux/reducers/OIDCReducer.js

```diff
@@ -127,7 +130,19 @@ export default function oidcReducer(state = INIT_STATE, action) {
       } else {
         return handleDefault()
       }
+    case GET_UMA_RESOURCES: 
```

Comment:
please create a separate reducer for uma resource.

Reply:
done, updated @duttarnab